package.json

```diff
@@ -9,13 +9,15 @@
   "files": [
     "bin/pixelmatch"
   ],
+  "engines": {
+    "node": ">=16"
+  },
```

Comment:
Let's drop the `engines` field — after all, the Node version requirement is only for development purposes, while the field will affect downstream users where Pixelmatch works perfectly fine on lower Node versions.